CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -0,0 +1,70 @@
+# Changelog 1.11
+
+## 1.11.0 - 2023-03-22
+
+- The EndpointSlice feature is graduated from Alpha to Beta and is therefore enabled by default.
+
+### Added
+
+- Add the following capabilities to Antrea-native policies:
+  * ClusterSet scoped policy rule now supports with the `namespaces` field. ([#4571](https://github.com/antrea-io/antrea/pull/4571), [@Dyanngg])
```

Comment:
ClusterSet-scoped policy rules now support the `namespaces` field

Reply:
done